flink-core/src/main/java/org/apache/flink/api/common/io/GlobFilePathFilter.java

```diff
@@ -52,8 +53,13 @@
 
 	private static final long serialVersionUID = 1L;
 
-	private final ArrayList<PathMatcher> includeMatchers;
-	private final ArrayList<PathMatcher> excludeMatchers;
+	private final List<String> includePatterns;
+	private final List<String> excludePatterns;
+
+	// Path matchers are not serializable so we are delaying their
+	// creation until they are used
+	transient private ArrayList<PathMatcher> includeMatchers;
```

Comment:
Now this is nitpicky, but i think the convention is that transient comes after private.